lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -1349,6 +1429,9 @@ def _extract_dwelltime_data_from_groups(groups, exclude_ambiguous_dwells):
             tracks which start in the first frame or end in the last frame of the kymograph are not
             used in the analysis, since the exact start/stop times of the binding event are not
             definitively known.
+        observed_minimum : bool
+            This is a legacy mode that uses the minimum dwelltime observed for each Kymo, rather
+            than the true minimum observable time.
```

Comment:
even though it's internal might be nice for our future selves to be explicit here
```suggestion
        observed_minimum : bool
            Whether to enable the legacy mode that incorrectly uses the minimum dwelltime observed for each Kymo, rather
            than the true minimum observable time.
```